### PR TITLE
form select fix

### PR DESCRIPTION
**Pull request by kowsar89**

### CODE REVIEW

**Comment by vairafiq** [comment]
looks good

File: templates/listing-form/fields/category.php [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
  */
 
-$placeholder = ! empty( $data['placeholder'] ) ? $data['placeholder'] : '';
-$multiple    = $data['type'] == 'multiple' ? 'multiple="multiple"' : '';
-$max         = !empty( $data['max'] ) ? 'data-max="'. $data['max'] .'"' : '';
-$create_new  = !empty( $data['create_new_cat'] ) ? ' data-allow_new="'. $data['create_new_cat'] .'"' : '';
+$placeholder = $data['placeholder'] ?? '';
+$data_max    = $data['max'] ?? '';
+$data_new    = $data['create_new_cat'] ?? '';
+$multiple    = $data['type'] == 'multiple' ? 'multiple' : '';
```
Comment (obiPlabon) [inline templates/listing-form/fields/category.php:11]: Please use strict equality `===`

File: templates/listing-form/fields/location.php [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
  */
 
-$placeholder = ! empty( $data['placeholder'] ) ? $data['placeholder'] : '';
-$multiple    = $data['type'] == 'multiple' ? 'multiple="multiple"' : '';
-$max         = !empty( $data['max_location_creation'] ) ? 'data-max="'. $data['max_location_creation'] .'"' : '';
-$create_new  = !empty( $data['create_new_loc'] ) ? ' data-allow_new="'. $data['create_new_loc'] .'"' : '';
+$placeholder = $data['placeholder'] ?? '';
+$data_max    = $data['max_location_creation'] ?? '';
+$data_new    = $data['create_new_loc'] ?? '';
+$multiple    = $data['type'] == 'multiple' ? 'multiple' : '';
```
Comment (obiPlabon) [inline templates/listing-form/fields/location.php:11]: Please use strict equality `===`

File: templates/listing-form/fields/tag.php [hunk 3]
```diff
@@ -2,22 +2,23 @@
...
  * @since   6.7
- * @version 7.3.1
+ * @version 7.3.3
  */
 
-$all_tags = get_terms( ATBDP_TAGS, array( 'hide_empty' => 0 ) );
+$placeholder = $data['placeholder'] ?? '';
+$data_max    = $data['max'] ?? '';
+$data_new    = $data['allow_new'] ?? '';
+$multiple    = $data['type'] == 'multiple' ? 'multiple' : '';
```
Comment (obiPlabon) [inline templates/listing-form/fields/tag.php:11]: Please use strict equality `===`

**Review by obiPlabon — CHANGES_REQUESTED** [review]

**Review by vairafiq — APPROVED** [review]
Looks good

**Review by obiPlabon — APPROVED** [review]

**Pull request merged**